arlo_server/routes.py

```diff
@@ -464,7 +465,7 @@ def audit_status(election_id=None):
             {
                 "id": j.id,
                 "name": j.name,
-                "contests": [c.contest_id for c in j.contests],
+                "contests": [c.id for c in j.contests],
```

Comment:
wow, was this just wrong code prior to your fix?

Reply:
No, it worked before. My database model change broke it